portfolio/src/main/webapp/style.css

```diff
@@ -223,3 +223,7 @@ footer {
 .extra-border-img{
   margin-top:12px;
 }
+
+#quote-container{
+    margin: 10px;
+}
```

Comment:
please add a new line at the of the file :) you can also delete the greeting-container as it's no longer being used

Reply:
Thanks, good spot!